### PR TITLE
[testing] add bloc-widget unit test

### PR DESCRIPTION
**Pull request by jonahwilliams**
- Adds an example of a unit test widget + bloc. 
- Makes some parameters required on the bloc to prevent leaving them off

### CODE REVIEW

File: test/screens/checkup_test.dart [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    expect(find.text("It's time for your checkup."), findsOneWidget);
+  });
+}
+
+Widget setUpCheckupScreen({
+  QuestionsBloc questions,
+  CheckupBloc checkup,
+  PreferencesBloc preferences,
+}) {
+  questions ??= QuestionsBloc(questionsRepository: QuestionsRepository());
```
Comment (jonahwilliams) [inline test/screens/checkup_test.dart]: If these blocs aren't setup exactly right, the bloc will NPE and that gets swallowed by the flutter_bloc package somewhere. We need to find a way to surface these errors so they crash the app or go to some error state

File: test/screens/checkup_test.dart [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    checkupsRepository: CheckupsRepository(),
+  );
+  preferences ??= PreferencesBloc();
+  return MaterialApp(
+    home: Scaffold(
+      body: BlocProvider(
+        child: BlocProvider(
+          child: BlocProvider(
+            child: CheckupScreen(),
+            create: (BuildContext context) => preferences,
```
Comment (goderbauer) [inline test/screens/checkup_test.dart]: uber-nit: move the 'create' params above the child params to make this slightly easier to read.

**Comment by jonahwilliams** [comment]
Filled https://github.com/coronavirus-diary/coronavirus-diary/issues/88 for the error issue

File: test/screens/checkup_test.dart (hunk 2 above)
Comment (jonahwilliams) [reply test/screens/checkup_test.dart]: Done!

**Review by goderbauer — DISMISSED** [review]
LGTM

File: test/screens/checkup_test.dart [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import 'package:covidnearme/src/blocs/checkup/checkup.dart';
```
Comment (goderbauer) [inline test/screens/checkup_test.dart]: Should this file be under test/**ui**/screens?

**Review by dnfield — APPROVED** [review]

File: test/screens/checkup_test.dart (hunk 3 above)
Comment (jonahwilliams) [reply test/screens/checkup_test.dart]: Fixed

**Pull request merged**